pcdsdevices/spectrometer.py

```diff
@@ -326,8 +341,19 @@ class TMOSpectrometer(BaseInterface, GroupDevice):
     transmission = 1
     SUB_STATE = 'state'
 
+    # dummy signal, state is always the same
+    lightpath_cpts = ['yag_x.user_readback']
 
-class HXRSpectrometer(BaseInterface, GroupDevice):
+    def calc_lightpath_state(self, **kwargs) -> LightpathState:
+        # TODO: get real logic here, instead of legacy hard-coding
```

Comment:
TODO reminder callout

Reply:
I skipped these devices because they weren't the big branching ones.  I'll make a followup PR when I get that logic sorted out.